c/chalice/chalice_ubi_8.3.sh

```diff
@@ -0,0 +1,81 @@
+# ----------------------------------------------------------------------------
+#
+# Package               : chalice
+# Version               : 1.22.4
```

Comment:
Is it not tested for any other versions ? If so, those versions also should be listed here.